notebooks/community/managed_notebooks/Fraud detection/Fraud Transaction Detection notebook.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>Don't hardcode this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>